README.md

```diff
@@ -52,8 +52,10 @@ RUNNER_TOKEN_SECRET_ARN="<ARN of your Runner Token Secret>" # This is outlined i
 PREFECT__CLOUD__AUTH_TOKEN="<A valid Prefect [Tenant token](https://docs.prefect.io/orchestration/concepts/tokens.html#tenant)> to support flow registration"
 PREFECT_PROJECT="<An existing Prefect [Project](https://docs.prefect.io/orchestration/concepts/projects.html#creating-a-project) where the bakery's test flows will be registered>"
 PREFECT_AGENT_LABELS="<A set of Prefect Agent [Labels](https://docs.prefect.io/orchestration/agents/overview.html#labels) which will be registered with the deployed agent to limit which flows should be executed by the agent>"
+BUCKET_USER_ARN="The manually created user whose credentials will be shared via bakeries.yaml"
```

Comment:
Just to be clear - this will be shared in the `THIS_IS_THE_ENV_VAR_NAME` style in bakeries.yaml correct? Where the actual arn is a GitHub Secret?

Reply:
The `BUCKET_USER_ARN` is only used by `pangeo-forge-aws-bakery`.  Many organizations (including our DevSeed policies) restrict the creation of new IAM Users so they can only be created by a User with Administrative rights and not with the User which performs the stack creation, so we need a mechanism to reference a manually created user.  

The credentials for this user are stored in `bakeries.yaml` https://github.com/pangeo-forge/bakery-database/blob/main/bakeries.yaml#L10 using the style you describe where the dictionary key references a secret that is stored at the Github repository or Organization level.